frontend/express/public/javascripts/countly/vue/core.js

```diff
@@ -372,6 +388,24 @@
         });
     };
 
+    var NotificationToastsView = VueCompositionAPI.defineComponent({
```

Comment:
Why composition API? :slightly_smiling_face:

Reply:
It's difficult to explain but my understanding was that I needed to render a vue component inside the core component (wrapper), meaning that any countlyVue element has not yet been initiated. I can tell that using composition API is on overkill but I really did not know what else would be the best approach to do it, plus I already saw that we were using the same interface with the dummy component. any suggestions?